ui/src/main/webapp/lib/ddf-sources/index.js

```diff
@@ -25,7 +24,7 @@ const CSW = {
   `,
   mutation: gql`
     mutation SaveCswSource($config: CswSourceConfiguration!){
```

Comment:
❓ Shouldn't we rename these mutations to `CreateXXXSource`?

And where are the `UpdateXXXSource` methods?